### PR TITLE
Update Markdown support for Approvals Teams app

### PR DESCRIPTION
**Pull request by mukhole**
Approvals Teams app currently does not support Markdown.
The documentation is not clearly stating this, causing confusion among users.
This change is to clearly state that the Approvals App on Teams does not currently support Markdown.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@mukhole : Thanks for your contribution! The author(s) have been notified to review your proposed change. @gcorvera

**Comment by PRMerger10** [comment]
@mukhole : Thanks for your contribution! The author(s) have been notified to review your proposed change. @gcorvera

File: articles/approvals-markdown-support.md [hunk 1]
```diff
@@ -30,21 +30,22 @@ This article teaches you how to use [Markdown](https://en.wikipedia.org/wiki/Mar
 >
 > - Approval request emails are *actionable messages*. If your [Microsoft Outlook client](https://docs.microsoft.com/outlook/actionable-messages/#outlook-version-requirements-for-actionable-messages) doesn't support actionable messages, it displays approval requests in HTML format. 
 > - All Markdown renderers have implementation differences. Review the [Client Support](#client-support) section for details.
+> - Markdown is not currently supported for Approvals app on Teams, but is expected soon.
```
Comment (dblyth-msft) [inline articles/approvals-markdown-support.md:33]: I'd say "in March 2021"

**Review by dblyth-msft — CHANGES_REQUESTED** [review]

**Comment by PRMerger8** [comment]
@MSFTMan : Thanks for your contribution! The author(s) have been notified to review your proposed change. @gcorvera

**Comment by PRMerger9** [comment]
@MSFTMan : Thanks for your contribution! The author(s) have been notified to review your proposed change. @gcorvera

**Review by MSFTMan — COMMENTED** [review]
@dblyth-msft I am not seeing the term FlowBot as an approved term, so I'll remove it from the PR. Kindly let me know if this gets approved and added to the style guide.

**Comment by PRMerger12** [comment]
@MSFTMan : Thanks for your contribution! The author(s) have been notified to review your proposed change. @gcorvera

**Review by MSFTMan — APPROVED** [review]

**Pull request merged**